Merge branch 'feature/#8' into dev

Jang-JIye opened the pull request:
Feature/#8 (User) 봉사활동 참여 신청

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

Jang-JIye committed:
Merge branch 'dev' into feature/#8

Jang-JIye committed:
Merge branch 'dev' into feature/#8

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

중간 커밋

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

중간 커밋

Jang-JIye committed:
Merge branch 'dev' into feature/#8

Jang-JIye committed:
#8 [refactor] (User) 봉사활동 참여 신청

Application -> Enrollment로 리펙토링
EnrollmentServiceImpl : 리펙토링

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

EnrollmentRequestDto, EnrollmentResponseDto

Jang-JIye committed:
Merge branch 'dev' into feature/#8

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

Enrollment , EnrollmentStatus 추가

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

controller, serviceImpl, repository : 봉사활동 참여 신청 / 신청 취소 기능 구현 중 오류 발생

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

Application은 실행되지만, H2-console에는 값이 들어오지 않음.

Jang-JIye committed:
Merge branch 'dev' into feature/#8

Jang-JIye committed:
Revert "#8 [feat] (User) 봉사활동 참여 신청"

This reverts commit b2f4dac64e9cb11f749c32a3466d27473c2eccd9.

Jang-JIye committed:
#8 [feat] (User) 봉사활동 참여 신청

봉사활동 참여 신청 / 신청취소 구현

Jang-JIye committed:
Merge branch 'dev' into feature/#8

elicho91 commented on src/main/java/com/team8/volunteerworkproject/service/EnrollmentService.java:
```diff
@@ -0,0 +1,17 @@
+    void participation(Long postId, String userId);
+*/
+    //참가 신청
+    EnrollmentResponseDto attend(Long postId, EnrollmentRequestDto requestDto, UserDetailsImpl userDetails);
```
Controller에서 userdetails.getUserId로 넘겨주셨으면 String userId로 수정해주세요~

elicho91 commented on src/main/java/com/team8/volunteerworkproject/service/EnrollmentService.java:
```diff
@@ -0,0 +1,17 @@
+    EnrollmentResponseDto attend(Long postId, EnrollmentRequestDto requestDto, UserDetailsImpl userDetails);
+
+    //참가 신청 취소(삭제)
+    void cancel(Long postId, UserDetailsImpl userDetails, Long enrollmentId);
```
Controller에서 userdetails.getUserId로 넘겨주셨으면 String userId로 수정해주세요~

Jang-JIye committed:
#8 [fix] (User) 봉사활동 참여 신청

코드 리뷰로 Controller : userdetails.getUserId로 넘김 -> Service : UserDetailsImpl userDetails -> userId 로 수정
